Logger/LogFactory.cs

```diff
@@ -1,10 +1,45 @@
-﻿namespace Logger;
+
+using System;
+
+namespace Logger;
+
 
 public class LogFactory
 {
-    public BaseLogger CreateLogger(string className)
+    private string? _logPath;
+    public string ConfigureFileLogger(string logPath)
     {
+        if (logPath == null)
+        {
+            throw new ArgumentNullException(nameof(logPath), " File path can not be null"); ;
+        }
+        else
+        {
+            this._logPath = logPath;
+            return _logPath;
+        }
+    }
 
-        return null;
+  
+
+    public BaseLogger? CreateLogger(string className)
+    {
+        if (_logPath == null)
+        {
+            throw new ArgumentNullException(nameof(className), " File path can not be null"); ;
+        }
+        else
+        {
+            
+            FileLogger logg = new (ConfigureFileLogger(_logPath!))
```

Comment:
Is this a spelling mistake?
```suggestion
            FileLogger log = new (ConfigureFileLogger(_logPath!))
```